src/lib/components/ratio/index.svelte

```diff
@@ -98,9 +98,10 @@
 
     @include config.between('sm-column-break', 'lg-page-break') {
       gap: var(--shim) var(--gutter);
+      // fixed width column to prevent layout shift as the ratio number changes
       grid-template:
         'heading number' min-content
-        'intro   intro' 1fr / auto 1fr;
+        'intro   intro' 1fr / auto var(--ratio-width);
```

Comment:
@mirisuzanne I noted it above the grid-template line originally, but do you think I need to be more explicit about that?

Reply:
oh, haha - yeah, that's good